src/descriptor/policy.rs

```diff
@@ -747,41 +744,39 @@ impl<Ctx: ScriptContext> ExtractPolicy for Miniscript<DescriptorPublicKey, Ctx>
             Terminal::Hash160(hash) => {
                 Some(SatisfiableItem::HASH160Preimage { hash: *hash }.into())
             }
-            Terminal::Multi(k, pks) => {
-                Policy::make_multisig(pks, Arc::clone(&signers), *k, false, secp)?
-            }
+            Terminal::Multi(k, pks) => Policy::make_multisig(pks, signers, *k, false, secp)?,
```

Comment:
There are a few spots where you've been using `&signers` and some others where you've used `signers`. Maybe for clarity it would be better to be more consistent?

Reply:
Updated. Changed `&signers` to `signers` where possible.